lerobot/templates/visualize_dataset_template.html

```diff
@@ -100,6 +100,14 @@ <h1 class="text-xl font-bold mt-4 font-mono">
             {% endfor %}
         </div>
 
+        <!-- Language instruction -->
+        {% if videos_info[0].language_instruction %}
+        <div class="text-sm font-bold mt-2">
+            Language Instruction: <em>{{ videos_info[0].language_instruction }}</em>
+        </div>
+        <hr class="my-4 border-gray-600">
```

Comment:
@mishig25 Does it look good to you? :)

![Screenshot_2024-08-28_at_10 46 22](https://github.com/user-attachments/assets/fa269091-4dcb-4cfd-ae30-b92be931cd8c)